source/triggers/groups/triggerGroup.js

```diff
@@ -15,8 +15,8 @@
 this.initialize = function() {
     this.triggers = [];
     this.canFire$ = [];
-    this.checkFrequency$ = 0.04 + ( Math.random() * 0.02 );
-    this.future( this.checkFrequency$ ).checkTriggers$();
+    this.checkFrequency$ = 0.08 + ( Math.random() * 0.02 );
+    // this.future( this.checkFrequency$ ).checkTriggers$();
```

Comment:
It is extremely mysterious to me why these would be causing a performance issue.  We have one trigger group with maybe a couple dozen triggers in it.  None of the checks that those triggers are doing are expensive.  This is maybe .01% of the amount of work that, say, rendering has to do - so what is making it so slow?

That said, this should be safe to check in - but eventually, we're going to need to understand why this is expensive and address it - it shouldn't even pop up on the radar.